site/content/contribute/server/developer-setup/osx.md

```diff
@@ -15,38 +15,38 @@
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
      
     
-2. Download and install homebrew: https://brew.sh/.
+3. Download and install homebrew: https://brew.sh/.
```

Comment:
Is the `brew install docker-compose` step missing here?

Reply:
@lieut-data Thanks for the review.

As stated in the [official documentation for docker-compose](https://docs.docker.com/compose/install):
> **Docker Desktop for Mac** and **Docker Toolbox** already include Compose along with other Docker apps, so Mac users do not need to install Compose separately.

I can confirm this on my Mac - I just installed Docker Desktop for Mac and docker-compose is already included:

```sh
$ docker-compose --version
docker-compose version 1.24.1, build 4667896b
```

Reply:
Ah, perfect! Should revert the section increment then to keep this as the second step?

Reply:
I think I misinterpreted the numbering a bit. I saw that in every file except the `osx.md` step number 2 was missing (jumping straight from step 1 to 3) and step number 2 in `osx.md` was `Download and install homebrew`, which of course is only necessary for MacOS. So I thought that step 2 _always_ is meant to be `Download and install homebrew` and the absence of step 2 in all other files was to indicate that brew is not to be installed.

So then when I added my step which was necessary before installing brew, I thought I had to reflect that by incrementing the numbering of the list.

Since that adds confusion, I could either add a new commit reverting all those numbering changes or close this PR and submit a new one without those numbering changes.

Reply:
I don't think there's any expectation of uniform numbering between the operating systems, so I'd recommend just pushing a new commit to revert that file. Thanks, @DropNib!

Reply:
Alright, thanks @lieut-data. I pushed the reversion.